doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -1,24 +1,95 @@
 [[my-services-tab]]
 
-== The My Services Tab
+== Services Tab
 
-The *My Services* tab lists all available services with a summary of information associated with each service, including *Service Name*, *Retirement Date*, *Number of VMs*, *Owner*, the date it was *Created On*, and its *Power State*. You can filter or sort services by those entities, and now also by *Relative Cost*. There is now the ability for users to perform power operations on virtual machines within the Self Service user interface using the new set of buttons, *Start*, *Stop*, and *Suspend* for each service.  
+The *Services* tab lists all available services with a summary of information associated with each service, including the *Service Name*, its current *Power State* represented by an icon, *Number of VMs*, *Owner*, date *Created*, and *Retires* date. Additionally, you can see the chargeback cost per service for a virtual machine's month-to-date usage. You can filter or sort services by those entities, and by *Relative Cost*. 
 
-image:SSUI_MyServices_tab.png[]
+On the same screen, you have the option to edit the service name and description, edit tags, set ownership, retire the service now or at a later date, or remove the service. You can also perform power operations on virtual machines by clicking more options _(represented by three dots)_ next to the service.
```

Comment:
[WIP] I added the word "button" so it's "the more options button", but this should still be reviewed.